src/formatDate.js

```diff
@@ -8,7 +8,40 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let year, month, day;
+  const parts = date.split(fromFormat[3]);
```

Comment:
Ensure that `fromFormat[3]` is indeed the separator used in the `date` string. If `fromFormat` does not have a separator at index 3, this will lead to incorrect splitting of the date string.